trino/trino.go

```diff
@@ -130,6 +130,8 @@ const (
 	trinoAddedPrepareHeader       = trinoHeaderPrefix + `Added-Prepare`
 	trinoDeallocatedPrepareHeader = trinoHeaderPrefix + `Deallocated-Prepare`
 
+	authorization = `Authorization`
```

Comment:
Isn't this supposed to be `Authentication`?

Reply:
No, this would be Authorization to match "Authorization: Bearer {token}"